src/docs/releases/2.1.0.md

```diff
@@ -32,11 +34,13 @@ Also, note the following updates in `ExternalLoginSettings`:
 - `UseScriptToSyncRoles` has been renamed to `ExternalLoginSettings.UseScriptToSyncProperties`.
 - `SyncRolesScript` has been renamed to `ExternalLoginSettings.SyncPropertiesScript`.
 
-!!! note
+!!! warning
     When updating recipes to configure `LoginSettings` or `RegistrationSettings`, ensure that the settings reflect the new class structure.
 
 ### User Registration Feature
 
+#### User Registration Feature Not Needed for External Authentication
```

Comment:
```suggestion
#### **User Registration** Feature No Longer Required for External Authentication
```

Reply:
Formatting with `**` won't show a difference in how the document site is rendered. Neither should we really try to change the formatting of titles one-off.

Reply:
right because it's already highlighted. updated.